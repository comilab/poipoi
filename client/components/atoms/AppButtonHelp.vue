<template lang="pug">
button(type="button")
  font-awesome-icon(:icon="faQuestionCircle")
</template>

<script lang="ts">
import { defineComponent } from 'nuxt-composition-api'
import { faQuestionCircle } from '@fortawesome/free-solid-svg-icons'

export default defineComponent({
  setup () {
    return {
      faQuestionCircle
    }
  }
})
</script>
