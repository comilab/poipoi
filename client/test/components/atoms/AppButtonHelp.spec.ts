import { render, RenderOptions } from '@testing-library/vue'

import AppButtonHelp from '~/components/atoms/AppButtonHelp.vue'

describe('components/atoms/AppButtonHelp', () => {
  let options: RenderOptions<AppButtonHelp>

  beforeEach(() => {
    options = {
      stubs: ['FontAwesomeIcon']
    }
  })

  test('renders correctly', () => {
    const { container } = render(AppButtonHelp, options)

    expect(container).toMatchSnapshot()
  })
})
