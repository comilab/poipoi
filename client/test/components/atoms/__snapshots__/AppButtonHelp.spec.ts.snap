// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/AppButtonHelp renders correctly 1`] = `
<div>
  <button
    type="button"
  >
    <fontawesomeicon-stub
      icon="[object Object]"
    />
  </button>
</div>
`;
